### PR TITLE
Add hint about libjpeg, libwebp, libavif, ...

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Fixed tickets | fixes #issuenum
| Related PRs | sulu/sulu#prnum
| License | MIT

#### What's in this PR?

Add hint about libjpeg, libwebp, libavif, ....

#### Why?

Images like alpine are may not shipping this libs.

### CODE REVIEW

File: bundles/media/imagine-adapter.rst [hunk 1]
```diff
@@ -10,6 +10,12 @@ Available adapters are:
 * `imagick_extension`_
 * `vips_extension`_
 
+.. note::
+
+    You may require to install on your operation system ``libjpeg-dev``, ``libpng-dev``, ``libwebp-dev``, ``libavif-dev``, ...
+    or other libraries to support the range of image mime types you want to support. Linux distributions like
+    Alpine may are not shipping them in the base image.
+
```
Comment (alexander-schranz) [inline bundles/media/imagine-adapter.rst]: @Prokyonn may you have better suggestions here to make this kind of things easer to understand.

File: bundles/media/imagine-adapter.rst [hunk 2]
```diff
@@ -47,8 +53,8 @@ To install it use following commands:
 
 .. code-block:: bash
 
-    apt-get install libmagickwand-dev inkscape
-    pecl install imagick
+    apt-get install libmagickwand-dev
+    apt-get install php8.3-imagick
```
Comment (alexander-schranz) [inline bundles/media/imagine-adapter.rst:58]: pecl is mostly not used for linux distributions.

File: bundles/media/imagine-adapter.rst [hunk 3]
```diff
@@ -71,7 +77,6 @@ To install it use following commands:
 .. code-block:: bash
 
     apt-get install libvips-dev
-    pecl install vips
```
Comment (alexander-schranz) [inline bundles/media/imagine-adapter.rst:74]: VIPS uses FFI and not longer requires a PHP extension: https://github.com/libvips/php-vips-ext

File: bundles/media/imagine-adapter.rst [hunk 4]
```diff
@@ -10,6 +10,12 @@ Available adapters are:
 * `imagick_extension`_
 * `vips_extension`_
 
+.. note::
+
+    You may require to install on your operation system ``libjpeg-dev``, ``libpng-dev``, ``libwebp-dev``, ``libavif-dev``, ...
+    or other libraries to support the range of image mime types you want to support. Linux distributions like
+    Alpine may are not shipping them in the base image.
```
Comment (Prokyonn) [inline bundles/media/imagine-adapter.rst]:
```suggestion
    You may need to install additional libraries on your operating system to support various image MIME types,
    such as ``libjpeg-dev, ``libpng-dev``, ``libwebp-dev``, ``libavif-dev``.
     
    Note that some Linux distributions, like Alpine, do not include these libraries in their base images by default.
```

**Review by Prokyonn — APPROVED** [review]

**Pull request merged**